src/FusedChannelData.ts

```diff
@@ -249,6 +250,10 @@ export default class FusedChannelData {
           mat.uniforms.outlierDrawMode.value = combination[i].feature?.outlierDrawMode;
           mat.uniforms.outOfRangeDrawMode.value = combination[i].feature?.outOfRangeDrawMode;
           mat.uniforms.hideOutOfRange.value = combination[i].feature?.hideOutOfRange;
+          // Offset IDs based on the current frame, for data without
+          // globally-unique IDs.
+          const idOffset = combination[i].feature?.timeToIdOffset[combination[i].time ?? 0] ?? 0;
+          mat.uniforms.idOffset.value = idOffset;
```

Comment:
might be worth assigning `combination[i].feature` to a temporary variable above to avoid some boilerplate